packages/hash/api/src/model/knowledge/org.model.ts

```diff
@@ -64,7 +64,8 @@ export default class extends EntityModel {
       );
     }
 
-    const { data: orgAccountId } = await graphApi.createAccountId();
+    const orgAccountId =
+      params.orgAccountId ?? (await graphApi.createAccountId()).data;
```

Comment:
Oh, interesting, do we just return the accountId as a raw string from `createAccountId`?

Reply:
Yes